.pre-commit-hooks.yaml

```
@@ -0,0 +1,15 @@
+# Copyright (C) 2023 Maxwell G <maxwell@gtmx.me>
+# SPDX-License-Identifier: GPL-3.0-or-later
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- id: lint-collection-docs
+  name: Lint collection docs
+  language: python
+  pass_filenames: false
+  entry: "antsibull-docs lint-collection-docs ."
+  # Default arguments based on the collection_template.
+  # Users can change these if they wish
+  args:
+    - "--plugin-docs"
+    - "--skip-rstcheck"
```

This should make sure ansible-core is installed:
```suggestion
    - "--skip-rstcheck"
  additional_dependencies:
    - ansible-core >= 2.14.0
```